### PR TITLE
Update korean #6c591cb

### PR DESCRIPTION
**Pull request by yunbj**
also add missed item in update #7797

### CODE REVIEW

File: PowerEditor/installer/nativeLang/korean.xml [hunk 1]
```diff
@@ -275,6 +275,12 @@
   <Item id="45011" name="UTF-8로 변환"/> 
   <Item id="45012" name="UCS-2 빅 엔디언으로 변환"/> 
   <Item id="45013" name="UCS-2 리틀 엔디언으로 변환"/> 
+  <Item id="45060" name="Big5 (중국어 번체)"/>
+  <Item id="45061" name="GB2312 (중국어 간체)"/>
+  <Item id="45064" name="OEM 861: 아이슬란드어"/>
```
Comment (xylographe) [inline PowerEditor/installer/nativeLang/korean.xml]: Typo, `id="45064"` ==> `id="45054"`

File: PowerEditor/installer/nativeLang/korean.xml [hunk 2]
```diff
@@ -1188,15 +1194,15 @@ langs.xml 파일을 복구하시겠습니까?"/>
   </Menus>
   </ProjectManager>
 <MiscStrings>
+<!-- $INT_REPLACE$ is a place holder, don't translate it -->
 <word-chars-list-tip value="이것을 통해 선택을 위해 더블 클릭할때 추가 문자를 현재 단어 문자에 포함하거나 &quot;전체 단어 일치&quot; 옵션을 체크해서 검색합니다 "/>
 <word-chars-list-warning-begin value="주의사항: "/>
-<!-- $INT_REPLACE$ is a place holder, don't translate it -->
```
Comment (donho) [inline PowerEditor/installer/nativeLang/korean.xml]: This is reminder so this comment should be kept

File: PowerEditor/installer/nativeLang/korean.xml [hunk 3]
```diff
@@ -1188,15 +1194,15 @@ langs.xml 파일을 복구하시겠습니까?"/>
...
 <word-chars-list-space-warning value="$INT_REPLACE$ 공간"/>
-<!-- $INT_REPLACE$ is a place holder, don't translate it -->
```
Comment (donho) [inline PowerEditor/installer/nativeLang/korean.xml]: This is reminder so this comment should be kept.

**Comment by donho** [comment]
You should create a new branch for each PR (whereas you use the old branch).
see : https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests

Please create a new branch for next PR.

**Pull request closed without merging**